### PR TITLE
Update reducers.md

### PR DESCRIPTION
**Pull request by comerc**

### CODE REVIEW

File: docs/guides/reducers.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ const fooActionCreator = () => ({
 
 
 // reducer
-const fooReducer = (state = {}, action) => {
+const fooReducer = (state = {}, action) => switch(action.type) {
```
Comment (notgiorgi) [inline docs/guides/reducers.md]: You cannot return `switch` from function, it's not an expression

**Comment by pburtchaell** [comment]
Your issue does not follow [the contributing guidelines](https://github.com/pburtchaell/redux-promise-middleware/blob/master/.github/CONTRIBUTING.md) for this project. Please review these guidelines, update your PR with a description, then reopen. Thank you!

**Comment by comerc** [comment]
WTF

**Comment by tomatau** [comment]
I think that might be automated. Just write a description and reopen. It looks good otherwise

**Comment by pburtchaell** [comment]
I only looked at the latest change and it wasn't clear from that commit that you added a switch statement. In the future, it does help to add a short description so I know what to expect in the changes. Thanks!

**Pull request merged**